packages/hooks/__tests__/use-throttle-render.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { defineComponent, nextTick, ref } from 'vue'
+import { mount } from '@vue/test-utils'
+import { afterEach, beforeEach, describe, expect, it } from 'vitest'
+import { useThrottleRender } from '../use-throttle-render'
+import type { VueWrapper } from '@vue/test-utils'
+
+const Comp = defineComponent({
+  setup() {
+    const loading = ref(false)
+    const throttled = useThrottleRender(loading, 1000)
+    // Test the settimeout branch clearly: trigger the watch to record the settimeout first, and then record the settimeout again when mount.
+    loading.value = true
+
+    return () => <div class="test-dom">{throttled.value.toString()}</div>
+  },
+})
+
+describe('useThrottleRender', () => {
+  let wrapper: VueWrapper<InstanceType<typeof Comp>>
+
+  beforeEach(() => {
+    wrapper = mount(Comp)
+  })
+
+  afterEach(() => {
+    wrapper.unmount()
+  })
+
+  it('should throttle rendering when loading is true', async () => {
+    await nextTick()
+    expect(wrapper.find('.test-dom').text()).toBe('false') // initially false
+    await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
There are related methods encapsulated internally and can be used directly.

https://github.com/element-plus/element-plus/blob/84f49de955848301b7c057bd443c69d83c3d2d1c/packages/test-utils/sleep.ts#L1

Reply:
> There are related methods encapsulated internally and can be used directly.
> 
> https://github.com/element-plus/element-plus/blob/84f49de955848301b7c057bd443c69d83c3d2d1c/packages/test-utils/sleep.ts#L1

ACK! and also, I found module redirection of `@element-plus/test-utils` is failed when i use sleep. I tried fix it in #16504